config.codekit

```diff
@@ -0,0 +1,1060 @@
+{
```

Comment:
remove this file from git and add to .gitignore.
this is only for local purpose, not used in project by other teammates, right?